src/igvfd/tests/test_schema_organoid.py

```diff
@@ -0,0 +1,41 @@
+import pytest
+
+
+def test_differentiation_dependency(organoid, testapp):
```

Comment:
If you end up using $merge to share dependency, you will only need the test code once as well.